packages/cli/src/commands/versions/bump.ts

```diff
@@ -49,7 +49,7 @@ const DEP_TYPES = [
   'optionalDependencies',
 ];
 
-const DEFAULT_PATTERN_GLOB = '@backstage/*';
+const DEFAULT_PATTERN_GLOB = '@backstage?(-community)/*';
```

Comment:
What are the risks around this given the community plugins don't release on the same schedule?

Reply:
I'm unsure. I don't think it's worse than what it is already? I've had backstage break both because plugins weren't updated and because they were. Though since we generally suggest that people keep everything updated, this default pattern seems like it would have the least amount of surprises.